pkg/controllers/generic/import_syncer.go

```diff
@@ -341,6 +372,11 @@ func (s *importer) TranslateMetadata(pObj client.Object) client.Object {
 	nn := s.PhysicalToVirtual(pObj)
 	vObj.SetName(nn.Name)
 	vObj.SetNamespace(nn.Namespace)
+
+	if s.syncerOptions.IsClusterScopedCRD {
+		vObj.SetLabels(nil)
+	}
```

Comment:
Q: What is the reason for removing the Labels?
I could imagine them being useful, e.g. resources that are exported might have LabelSelectors that would be selecting on these cluster-scoped imported resources. AFAIK Prometheus uses this.

Reply:
> * Only support 1:1 name mappings without any label selectors  

ref: point 1 from ENG-887

Reply:
Syncing labels on the resources is not the same as syncing based on the labels. This code is not related to label selectors as far as I can tell.